src/ripple/rpc/impl/ServerHandlerImp.cpp

```diff
@@ -377,6 +377,27 @@ ServerHandlerImp::onStopped(Server&)
 
 //------------------------------------------------------------------------------
 
+template <class T>
+void
+logDuration(
+    Json::Value const& request,
+    T const& duration,
+    beast::Journal& journal)
+{
+    auto const seconds =
+        std::chrono::duration_cast<std::chrono::seconds>(duration).count();
+
+    auto const level = (seconds >= 10)
+        ? journal.error()
+        : (seconds >= 1) ? journal.warn() : journal.debug();
```

Comment:
This can be significantly simplified:
```
    using namespace std::chrono_literals;
    auto const level = (duration >= 10s)
        ? journal.error()
        : (duration >= 1s) ? journal.warn() : journal.debug();
```